packages/gatsby-plugin-pixel-facebook/src/pixel/handler.ts

```diff
@@ -49,11 +49,11 @@ export const handler: PixelEventHandler = (event) => {
             acc + (typeof price === 'number' ? price : 0),
           0
         ),
-        content_ids: items.map((sku) => sku.id),
-        contents: items.map((sku) => ({
-          id: sku.id,
-          quantity: sku.quantity,
-          item_price: sku.price ? sku.price : 0,
+        content_ids: items.map((product) => product.skuId),
```

Comment:
items are skus, aren't they?

Reply:
They are not. They are the items of the event, and now that you pointed that out, I think maybe we should rename it to `products`. What do you think?